### ISSUE TITLE
IcingaDB-web is spamming "deprecated" messages on debian bookworm (php8.2)

### ISSUE THREAD

**Issue by mafri27** [Icinga/icingadb-web#780]
## Describe the bug

Messages like

```
Deprecated
: Creation of dynamic property ipl\Sql\Config::$type is deprecated in
/usr/share/icinga-php/ipl/vendor/ipl/sql/src/Config.php
on line 32
```
## To Reproduce

install icingadb-web on debian12

## Expected behavior
no errors

## Screenshots
![image](https://github.com/Icinga/icingadb-web/assets/47006694/db3b6b76-d469-4919-9f73-3ade6421bd2a)

## Your Environment
```

icingacli --version
PHP Deprecated:  Using ${var} in strings is deprecated, use {$var} instead in /usr/share/icingaweb2/application/clicommands/VersionCommand.php on line 50

Deprecated: Using ${var} in strings is deprecated, use {$var} instead in /usr/share/icingaweb2/application/clicommands/VersionCommand.php on line 50
PHP Deprecated:  Using ${var} in strings is deprecated, use {$var} instead in /usr/share/icingaweb2/application/clicommands/VersionCommand.php on line 52

Deprecated: Using ${var} in strings is deprecated, use {$var} instead in /usr/share/icingaweb2/application/clicommands/VersionCommand.php on line 52
PHP Deprecated:  Creation of dynamic property Icinga\Clicommands\VersionCommand::$trace is deprecated in /usr/share/php/Icinga/Cli/Command.php on line 61

Deprecated: Creation of dynamic property Icinga\Clicommands\VersionCommand::$trace is deprecated in /usr/share/php/Icinga/Cli/Command.php on line 61
Unable to locate monitoring module
Icinga Web 2  2.11.4    
PHP Version   8.2.7     
MODULE    VERSION   
grafana   2.0.3     
icingadb  1.0.2  
```

**Comment by Rolf-Zi** [comment]
I got the gui  working with the following hack. But I do not recommend to use this in production:

```
sudo sed -i '/class Web/i #[\\AllowDynamicProperties]' /usr/share/php/Icinga/Application/Web.php
sudo sed -i '/class Config/i #[\\AllowDynamicProperties]' /usr/share/icinga-php/ipl/vendor/ipl/sql/src/Config.php
```

**Comment by nilmerg** [comment]
Will be solved with v1.1
